dev_docs/tutorials/sample_data.mdx

```diff
@@ -0,0 +1,33 @@
+---
+id: kibDevTutorialSampleData
+slug: /kibana-dev-docs/tutorial/sample-data
+title: Add sample data
+summary: Learn how to add sample data to Kibana
+date: 2021-04-26
+tags: ['kibana', 'onboarding', 'dev', 'architecture', 'tutorials']
+---
+
+## Installation from the UI
+
+1) Navigate to the home page
```

Comment:
Use "1." instead of "1)".  Also, add an ending period to each step.

1. Navigate to the home page.

Reply:
Pushed with c98f1d5